src/components/LoginUser.js

```diff
@@ -50,7 +50,7 @@ function LoginUser() {
                 </form>
                 {loginMsg && <p>Successfully logged in!</p>}
                 {userValue ?
-                    <button className="loginBtn btn btn-success"
+                    <button className="loginBtn btn btn-danger"
```

Comment:
don't use the danger css, use the happy css

Reply:
Sounds good